common/capabilities/channel.go

```diff
@@ -67,7 +67,7 @@ func (cp *ChannelProvider) HasCapability(capability string) bool {
 	switch capability {
 	// Add new capability names here
 	case ChannelV3_0:
-		return false // Will be enabled in future v3.x binaries
```

Comment:
I know that we'd intentionally set this as false, but this breaks a number of unit tests because they are using the sample config that specifies v3 channel capability.